GTG/gtk/editor/editor.py

```diff
@@ -164,7 +174,7 @@ def __init__(self,
             self.task.set_to_keep()
         self.textview.modified(full=True)
         self.window.connect("destroy", self.destruction)
-        self.calendar.connect("date-changed", self.on_date_changed)
+        # self.calendar.connect("date-changed", self.on_date_changed)
```

Comment:
Can you add todo for this? Either provide the functionality or drop it completely?